tensorflow/lite/micro/tools/make/helper_functions.inc

```diff
@@ -303,10 +321,12 @@ endef
 # generate the standalone project.
 define generate_microlite_projects
 $(call generate_project,make,$(MAKE_PROJECT_FILES),$(1),$(MICROLITE_CC_SRCS) $(THIRD_PARTY_CC_SRCS) $(2),$(MICROLITE_CC_HDRS) $(THIRD_PARTY_CC_HDRS) $(MICROLITE_TEST_HDRS) $(3),$(LDFLAGS) $(MICROLITE_LIBS),$(CXXFLAGS) $(GENERATED_PROJECT_INCLUDES), $(CCFLAGS) $(GENERATED_PROJECT_INCLUDES))
+ifneq ($(TARGET_ARCH), arc)
```

Comment:
Can you remove this ifneq? We'd prefer not to have target-conditional logic at this level.

Reply:
done